app/helpers/my_modules_helper.rb

```diff
@@ -31,7 +31,7 @@ def az_ordered_assets_index(step, asset_id)
 
   def get_task_alert_color(my_module)
     alert = ''
-    unless my_module.completed?
+    unless my_module.archived? || my_module.completed?
```

Comment:
Isn't there an unwritten rule that unless should be done only when you have one condition, otherwise 'if' should be used?

Reply:
Linter doesn't say anything about it

Reply:
As said, it's a good practice, changing it with if makes it more readable. Change it please.